avocado_vt/plugins/vt.py

```diff
@@ -49,6 +49,70 @@
                            "plugin to get rid of this message")
 
 
+def add_basic_vt_options(parser):
+    """
+    Add basic vt options to parser
+    """
+    parser.add_argument("--vt-config", action="store", dest="vt_config",
+                        help="Explicitly choose a cartesian config. When "
+                        "choosing this, some options will be ignored (see "
+                        "options below)")
+    parser.add_argument("--vt-type", action="store", dest="vt_type",
+                        help="Choose test type (%s). Default: qemu"
+                        % ", ".join(SUPPORTED_TEST_TYPES), default='qemu')
```

Comment:
I know this is being copied over, but we should be using: "Default: %(default)s" instead to not repeat ourselves.